src/routes/__tests__/healthcheck.test.ts

```diff
@@ -0,0 +1,44 @@
+const mockGet = jest.fn();
+jest.mock('express', () => {
+	return () => ({
+		get: mockGet
+	});
+});
+
+import healthcheck from '../healthcheck';
+import * as express from 'express';
+
+describe('/healthcheck route', () => {
+	const responseMock = {
+		status: jest.fn(() => responseMock),
+		send: jest.fn()
+	} as unknown as express.Response;
+
+	it('sets response status code as 200', () => {
```

Comment:
Forgot to add a test to check that the endpoint is actually added to the express routing 🙃

Reply:
Done